src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageSolutionDetailControlModel.cs

```diff
@@ -81,8 +81,13 @@ private void UpdateInstalledVersions()
         {
             var hash = new HashSet<NuGetVersion>();
 
+            IsRequestedVisible = false;
             foreach (var project in _projects)
             {
+                if (IsRequestedVisible.Equals(false) && project.NuGetProject.ProjectStyle.Equals(NuGet.ProjectModel.ProjectStyle.PackageReference))
+                {
+                    IsRequestedVisible = true;
```

Comment:
use local here if you agreed with my suggestion

Reply:
Was the decision to display the request column whenever a PR project is available? 
I thought it was only when *all* projects are PR.